book/browsersandweb.md

```diff
@@ -60,91 +60,90 @@ and fun journey. That's what this book is about.
 Explaining the black box
 ========================
 
-As you may already know from making websites, the core implementation components
-of the web are approachable enough - an HTML & CSS-based document model, HTTP,
-hyperlinks, and JavaScript. Most people can learn easily enough how to make simple
-HTML pages; programming abilities are not required. But how does the browser
-actually do its job of rendering that HTML? As it turns out, not many people
-who don’t work on browsers actually know in much detail, even trained software
-developers![^software-developers]
-
-[^software-developers]: I usually prefer the word “engineer”, but on the web
-it’s much more common to use the word “developer”, or more specifically “web
-developer”, so I’ll use that term in this book. An additional advantage of using
-“developer” is that it’s not necessary to be a trained software engineer or
-computer scientist to build websites; on the contrary, one goal of the web has
-always been  to make its use accessible to all people, not just experts. In fact
-many websites are in large part built by those trained in other disciplines;
-“web developer” is more inclusive of these additional, critical roles.
-
-Most of us developers treat the browser as a black box, one that is either
-magical or frustrating (depending on whether it is working correctly or not!).
-After all, HTML & CSS _are_ black boxes, or more precisely declarative APIs -
-ones in which one specifies _what_ outcome is desired, as opposed to _how_ to
-achieve that outcome. It’s the _browser itself_ that is responsible for figuring
-out the how. Not only are website developers encouraged not to say how exactly
-the pixels on the screen are generated, in most cases there _is no feasible way_
-for developers to draw their website’s pixels “on their own”.
-
-In that sense, they also lose control and some amount of agency---when those
-pixels are wrong, they cannot directly fix them.[^loss-of-control] However, this
-loss of control comes with powerful upsides, such as: it’s much easier to make
-and deploy content on the web without having to implement many of the details;
-that content is instantly (magically!) available on every computing device in
-existence; and the content is likely to be accessible in the future, avoiding
-(for the most part) the inevitable obsolescence of most software.
-
-[^loss-of-control]: Loss of control not necessarily specific to the web - much
+HTML, CSS, HTTP, hyperlinks, and JavaScript---the core of the web---are
+approachaable enough, and if you've made a website before[^recommended] you've
+seen that programming ability is not required. But not many people---not even
+professional software developers---know much about how a browser renders web
+pages![^software-developers]
+
+[^recommended]: If you haven't, I recommend MDN's [Learn Web
```

Comment:
Move these footnotes to the very top of the chapter where we say what "the web" is.